wcomponents-core/src/main/java/com/github/bordertech/wcomponents/util/HtmlSanitizer.java

```diff
@@ -0,0 +1,100 @@
+package com.github.bordertech.wcomponents.util;
```

Comment:
Might not seem necessary, but a bit of a naming convention we have is that any "util" classes are suffixed with "Util".